### ISSUE TITLE
Data types :date and :datetime

### ISSUE THREAD

**Issue by anka** [koss-lebedev/active_dynamic#9]
Really thanks for your nice gem and work done already. I would like to ask if it is possible to use the gem with `date` and `datetime` attributes to. Does it support this data types already and save/restore it correctly within the database? Do you have any examples for this use case?

**Comment by Svashta** [comment]
Dynamic attribute values are always stored in **text** value field, so you can't have native date and datetime fields. However, what you store as a value is up to you - so date and datetime values are certainly possible and i have successfully implemented both in my case.

It is the datatype + view definitions combination that you need.
I use a normal string field with jquery datetimepicker, so it is restricted to date or datetime values and works perfectly.

I don't have any advanced search capabilites implemented for dynamic attrbutes, but as far as i have tried it can be done well enough with elasticsearch.

So, in short, all the dynamic attribute values are stored in text field, but you can easily do advanced stuff with it. In my case i have strings, texts, booleans, dates and times etc... Everything.
